dev-scripts/quick

```diff
@@ -11,7 +11,7 @@ TAG="${TAG:-$(yq '.env.TAG | sub("-.*", "")' < .github/workflows/pull-request.ym
 OS="${OS:-linux}"
 ARCH="${ARCH:-amd64}"
 REPO="${REPO:-rancher}"
-CATTLE_K3S_VERSION=$(grep -m1 'ENV CATTLE_K3S_VERSION' package/Dockerfile | awk '{print $3}')
+CATTLE_K3S_VERSION=$(grep -m1 'ENV CATTLE_K3S_VERSION=' package/Dockerfile | cut -d '=' -f2)
 CATTLE_KDM_BRANCH=$(grep -m1 'ARG CATTLE_KDM_BRANCH=' package/Dockerfile | cut -d '=' -f2)
 CATTLE_RANCHER_WEBHOOK_VERSION=$(yq '.webhookVersion' < build.yaml)
 CATTLE_REMOTEDIALER_PROXY_VERSION=$(yq '.remoteDialerProxyVersion' < build.yaml)
```

Comment:
One day I'll look into having a single place to put values.. Feels like we should have a .env file instead of finding _some_ values in dockerfile and some other in build.yaml, etc. (Not to mention the use of yq makes me mad everytime I'm on a new machine.. which is not too often, but it gets me anyway lol)

Reply:
100% agree. Do we have any specific reason to use a `yaml` file?